docs/changes.rst

```diff
@@ -23,6 +23,33 @@ information to the contrary.
 
 
 ------------------
+3.9.0 - 2017-05-14
+------------------
+
+This is a feature release, adding datetime-related strategies to the core strategies.
+
+``extra.datetime.timezones`` allows you to sample pytz timezones from
+the Olsen database.  Use directly in a recipe for tz-aware datetimes, or
+compose with `st.none()` to allow a mix of aware and naive output.
+
+The new ``dates``, ``times``, ``datetimes``, and ``timedeltas`` strategies
+in ``hypothesis.strategies`` are all constrained by objects of their type.
+This means that you can generate dates bounded by a single day
+(i.e. a single date), or datetimes constrained to the microsecond.
+
+``times`` and ``datetimes`` take an optional ``timezones=`` argument, which
+defaults to ``none()`` for naive times.  You can use our extra strategy
+based on pytz, or roll your own timezones strategy with dateutil or even
+the standard library.
+
+The old ``dates``, ``times``, and ``datetimes`` strategies in
+``hypothesis.extra.datetimes`` are deprecated in favor of the new
+core strategies, which are more flexible and have no dependencies.
+
+The default field mapping for DateTimeField in the Django extra now respects
+the ``USE_TZ`` setting when choosing a strategy.
```

Comment:
Worth referencing #439 here?

Also, I feel like I should ask you to separate this out into its own patch release, but I won't actually insist on it.